### DISCUSSION TITLE
有没有大佬看一下哪里出错了

### DISCUSSION THREAD

**Discussion by eewwewewefs** [Meloong-Git/PCL#3856] [提问]
1.18.2的我的世界，加了大约四十来个mod，启动时出错，显示optifine和foroge冲突了，求大佬帮我找一下原因
![Uploading QQ截图20240519112848.png…]()
[Uploading 错误报告-2024-5-19_11.12.01.zip…]()

**Comment by MoYuan-CN** [comment]
你的任何一个文件都没有上传上来……
请等待 `Uploading XXX` 字样消失后再发送
另，你可以使用右上角 `...` 中的 `Edit` 来修改你已经发送的内容

**Comment by eewwewewefs** [comment]
补一下报告
[错误报告-2024-5-19_11.12.01.zip](https://github.com/Hex-Dragon/PCL2/files/15367473/-2024-5-19_11.12.01.zip)
![QQ截图20240519112848](https://github.com/Hex-Dragon/PCL2/assets/149159021/462f84a7-4881-4211-b7d4-00c2a8fd3e05)

**Reply by MoYuan-CN** [reply]
呃……看起来就是不兼容导致的
你打开 https://optifine.net/downloads 对着看你的 Forge 版本需要用哪个 OptiFine

**Comment by eewwewewefs** [comment]
不对啊，同一个配置，没加mod就可以运行，为什么啊？
![2](https://github.com/Hex-Dragon/PCL2/assets/149159021/4178e522-6ebf-47b2-b0aa-9a0a835a77c4)
![1](https://github.com/Hex-Dragon/PCL2/assets/149159021/ec501c73-990a-4f87-95da-b939c866080e)
![3](https://github.com/Hex-Dragon/PCL2/assets/149159021/badaf1eb-ae02-486d-b5a0-b4280573d332)

**Comment by alazeprt** [comment]
永恒枪械工坊（TAC）模组出现mixin注入问题

**Reply by MoYuan-CN** [reply]
不对，貌似是这个模组导致的
![image](https://github.com/Hex-Dragon/PCL2/assets/109946426/dc4b154c-1363-4cb6-814d-cb1d95163ba0)

**Reply by eewwewewefs** [reply]
mod列表没有这个mod啊
![6](https://github.com/Hex-Dragon/PCL2/assets/149159021/7a4f093d-fbff-42f8-b602-9ead309e3b11)

**Reply by alazeprt** [reply]
列表里tac开头的删了

**Comment by hejiehao** [comment]
应该是 OptiFine 和 tac 不兼容

**Reply by eewwewewefs** [reply]
好像不是吧
![5](https://github.com/Hex-Dragon/PCL2/assets/149159021/ba8b0d09-c6b1-40f6-a2cd-88e8704e9b43)

**Reply by hejiehao** [reply]
因为你没加 tac

**Comment by eewwewewefs** [comment]
永恒枪械工坊（TAC）的mixin能不能修改或者进行修复，有大佬知道怎么做吗

**Comment by eewwewewefs** [comment]
或者有没有可以平替的的旧版本
